### PR TITLE
[Issue-1231] Fix helm artifact extension to "*.tgz"

### PR DESCRIPTION
**Pull request by fidget83**
#1231

### CODE REVIEW

**Comment by fidget83** [comment]
Maybe add the artifact extension as plugin parameter with default value "tar.gz" for backward compatibility?

File: doc/src/main/asciidoc/inc/goals/build/_fabric8-helm.adoc [hunk 1]
```diff
@@ -59,6 +59,10 @@ This configuration section know the following subelements in order to configure
 | *engine*
 | The template engine to use
 |
```
Comment (rohanKanojia) [inline doc/src/main/asciidoc/inc/goals/build/_fabric8-helm.adoc]: Could you add `fabric8.helm.chartArtifactType` also to asciidoc, please?

Comment (fidget83) [reply doc/src/main/asciidoc/inc/goals/build/_fabric8-helm.adoc]: I'm sorry, but could you please provide a path to file? I cannot find it at all :(

Comment (rohanKanojia) [reply doc/src/main/asciidoc/inc/goals/build/_fabric8-helm.adoc]: I'm talking about the variable you've added in this PR

**Pull request merged**